src/index.ts

```diff
@@ -212,22 +208,55 @@ const tracker: JupyterFrontEndPlugin<IDebugger> = {
       update;
     });
 
-    const command = CommandIDs.create;
+    const commandCreateDebugger = CommandIDs.create;
+
+    const commandStartDebugger = CommandIDs.start;
+
+    const commandStopDebugger = CommandIDs.stop;
+
+    var commandStop: IDisposable;
+
+    const getModel = () => {
+      return tracker.currentWidget ? tracker.currentWidget.content.model : null;
+    };
+
+    app.commands.addCommand(commandStopDebugger, {
```

Comment:
How about directly using `CommandIDs.stop` here? (same question for the others commands)